### PR TITLE
Amber20

### PR DESCRIPTION
**Pull request by orbsmiv**
For INC1033241

Requires https://github.com/bear-rsg/easybuild-easyblocks/pull/63

* [x] Assigned to reviewer

`Amber-20.9-AmberTools-20.14-foss-2020a-Python-3.8.2.eb`:
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] EL7.9-cascadelake
* [ ] EL7.9-haswell
* [ ] EL8-cascadelake
* [ ] EL8-haswell
* [ ] EL8-power9

Add these if requested:
`Amber-20.9-AmberTools-20.14-fosscuda-2020a-Python-3.8.2.eb`:
* [ ] EL8-haswell
* [ ] EL8-power9

### CODE REVIEW

**Comment by branfosj** [comment]
closing / reopening to rerun the test

File: easybuild/easyconfigs/a/Amber/Amber-20.9-AmberTools-20.14/tell_amber_about_ppc64le.patch [hunk 1]
```diff
@@ -0,0 +1,56 @@
+Allow AmberTools to detect a ppc64le is and tell it what to do when it detects one
```
Comment (branfosj) [inline easybuild/easyconfigs/a/Amber/Amber-20.9-AmberTools-20.14/tell_amber_about_ppc64le.patch:1]: Patch is not used - so delete

File: easybuild/easyconfigs/a/Amber/Amber-20.9-AmberTools-20.14-fosscuda-2020a-Python-3.8.2.eb [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    ('PnetCDF', '1.12.1'),
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', '-Python-%(pyver)s'),  # mpi4py required for MMPBSA
+    ('Tkinter', '3.8.2'),
+    ('X11', '20200222'),
+    ('zlib', '1.2.11'),
+]
+
+# Numerous tests fail, which is expected by Amber devs. Need to handle this with a
+# threshold-based strategy in the easyblock
```
Comment (branfosj) [inline easybuild/easyconfigs/a/Amber/Amber-20.9-AmberTools-20.14-fosscuda-2020a-Python-3.8.2.eb:57]:
```suggestion
# Different tests pass and fail when using different numbers of processes. The manual makes it clear that this is expected.
```

File: easybuild/easyconfigs/a/Amber/Amber-20.9-AmberTools-20.14-foss-2020a-Python-3.8.2.eb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    ('PnetCDF', '1.12.1'),
+    ('Python', '3.8.2'),
+    ('SciPy-bundle', '2020.03', '-Python-%(pyver)s'),  # mpi4py required for MMPBSA
+    ('Tkinter', '3.8.2'),
+    ('X11', '20200222'),
+    ('zlib', '1.2.11'),
+]
+
+# Numerous tests fail, which is expected by Amber devs. Need to handle this with a
+# threshold-based strategy in the easyblock
```
Comment (branfosj) [inline easybuild/easyconfigs/a/Amber/Amber-20.9-AmberTools-20.14-foss-2020a-Python-3.8.2.eb:56]:
```suggestion
# Different tests pass and fail when using different numbers of processes. The manual makes it clear that this is expected.
```

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/a/Amber/Amber-20.9-AmberTools-20.14/tell_amber_about_ppc64le.patch (hunk 1 above)
Comment (orbsmiv) [reply easybuild/easyconfigs/a/Amber/Amber-20.9-AmberTools-20.14/tell_amber_about_ppc64le.patch]: resolved in 716f7d9

**Review by branfosj — APPROVED** [review]

**Pull request merged**